xbmc/pictures/PictureInfoTag.cpp

```diff
@@ -1033,7 +1033,7 @@ void CPictureInfoTag::SetInfo(int info, const std::string& value)
     }
   case SLIDE_EXIF_DATE_TIME:
     {
-      strcpy(m_exifInfo.DateTime, value.c_str());
+      strncpy(m_exifInfo.DateTime, value.c_str(), sizeof(m_exifInfo.DateTime) - 1);
```

Comment:
added the explicit null termination, to not get in trouble if something gets changed in the future.